source/sphinx_extensions/plugin_directory/extension.py

```diff
@@ -32,47 +42,41 @@ def generate_rst(app):
         fh.write(rendered)
 
     for plugin in plugins.values():
-        plugin_dir = os.path.join(rst_dir, plugin.name)
+        plugin_cli_name = plugin.name.replace('_', '-')
```

Comment:
I updated the code and templates to use CLI-style names (i.e. dashed names) for plugins. Plugins are typically named with dashes already (e.g. `'feature-table'` for `Plugin.name`) but that is by convention only, so there could be non-dashed names in the wild. This change is mainly to future-proof the code.

Action names are also converted to CLI-style names (see below).